ui/00_message/common/item_name.xml

```diff
@@ -19659,7 +19659,7 @@
   </item>
   <item id="7839">
     <original>鬼どぶろく</original>
-    <translation>Demon Unrefined Wine</translation>
+    <translation>Unrefined Demon Wine</translation>
```

Comment:
I'm certain that's a grammatical error. If that's how the White Dragon Temple translated it, they mistranslated it.